code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1231,3 +1231,26 @@ datum/reagent/romerol
 	// Silently add the zombie infection organ to be activated upon death
 	new /obj/item/organ/body_egg/zombie_infection(H)
 	..()
+
+/datum/reagent/growthserum
+	name = "Growth serum"
+	id = "growthserum"
+	description = "A commercial chemical designed to help older men in the bedroom."//not really it just makes you a giant
+	color = "#ff0000"//strong red. rgb 255, 0, 0
+	var/current_size = 1
+
+/datum/reagent/growthserum/on_mob_life(mob/living/carbon/human/H)
```

Comment:
Why are you typecasting a reagent to humans only when this can impact carbons *and* you don't do an `ishuman` check?